manifests/p/prefix-dev/pixi/0.9.1/prefix-dev.pixi.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created with Komac v1.9.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: prefix-dev.pixi
+PackageVersion: 0.9.1
+InstallerLocale: en-US
+InstallerSwitches:
+  Custom: -norestart
+UpgradeBehavior: install
+ReleaseDate: 2023-11-29
+AppsAndFeaturesEntries:
+- DisplayName: pixi
+  Publisher: pixi contributors
+  UpgradeCode: '{D983A0B8-0183-4F99-BA13-A1C49BD8E46F}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/prefix-dev/pixi/releases/download/v0.9.1/pixi-x86_64-pc-windows-msvc.msi

```

Comment:
There is also a `.exe` provided. Which is the preferred way for winget?
AFAIK, the msi file just extracts the executable somewhere, right @wolfv?

Reply:
I think @baszalmstra knows that better (he made the MSI installer)

Reply:
The exe is the straight up pixi binary. The MSI installs that executable into `~/.pixi/bin` and sets up some PATH variables. I guess we need the `msi` here.

Reply:
not related to this PR, but personally I would prefer installing to a dir that often already exists in PATH instead of adding a new one for a single .exe. (Nominations: `~/.local/bin`, `~/bin`)

Reply:
Those don't exist on Windows.

Reply:
Folders under %Appdata% are usually good candidates. Maybe `C:\Users\{username}\AppData\Local\Utilities`